packages/opentelemetry-web/src/WebTracer.ts

```diff
@@ -14,25 +14,51 @@
  * limitations under the License.
  */
 
-import { BasicTracer, BasicTracerConfig } from '@opentelemetry/tracing';
+import { BasePlugin } from '@opentelemetry/core';
+import {
+  BasicTracer,
+  BasicTracerConfig,
+  SpanProcessor,
+} from '@opentelemetry/tracing';
 import { StackScopeManager } from './StackScopeManager';
 
+/**
+ * WebTracerConfig provides an interface for configuring a Web Tracer.
+ */
+export interface WebTracerConfig extends BasicTracerConfig {
+  /**
+   * span processor
+   */
+  spanProcessor?: SpanProcessor;
```

Comment:
This is not getting used anywhere. On top of that, we have `addSpanProcessor` method in the tracer, we should rely on that.